spicy_api.py

```diff
@@ -78,103 +82,208 @@ def build_publisher_name(token):
     return f"pub_{token}"
 
 
+# Temporary, we shouldn't keep this long term
 @spicy_api.route("/get_full_database", methods=['GET'])
 def index():
     return FIREBASE_OBJ.get_data()
 
 
-@spicy_api.route("/get_vehicle_id", methods=['POST'])
-def user_info():
+# Revoke a token (Should be done on logout)
+@spicy_api.route("/revoke_token", methods=['POST'])
+def revoke_token():
+    """
+    post_request sample: {
+        "token": "token",
+    }
+
+    :return:
+    """
+    post_request = request.json
+    try:
+        user = get_user(post_request['token'])
+    except KeyError:
+        return "Error: Missing login token."
+    # TODO Revoke the token
+    # Return an empty json object to indicate success
+    return {}
+
+
+# Verify that your token is still valid
+@spicy_api.route("/verify_token", methods=['POST'])
+def verify_token():
+    """
+    post_request sample: {
+        "token": "token",
+    }
+
+    :return:
+    """
+    post_request = request.json
+    try:
+        user = get_user(post_request['token'])
```

Comment:
Only have token = post_request['token'] in the try except and don't call get_user() in there. It could cause things to work in a strange and unpredictable way.